submissions/AntonKottans/html-css-popup/index.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="stylesheet" href="style.css" />
+		<title>Popup window</title>
+	</head>
+	<body>
+		<header>
+			<nav>
+				<ul class="nav-menu">
+					<li class="nav-menu-item">
+						<a class="nav-menu-item-link" href="#">Mail</a>
+					</li>
+					<li class="nav-menu-item">
+						<a class="nav-menu-item-link" href="#">Images</a>
+					</li>
+					<li class="nav-menu-item">
+                        <input
+                        type="checkbox"
+                        class="popup-checkbox hidden-checkbox"
+                        id="popup-checkbox"
+                    />
+						<label for="popup-checkbox" class="popup-checkbox-label">
+							<img class="nav-image" src="img/popup-button.png" alt="" />
+						</label>
+						<div class="popup-menu">
+							<a href="#" class="popup-item"
+								><img
+									class="popup-item-img"
+									alt=""
```

Comment:
Each attribute `alt` should contains a text, it is necessary for [a11y](https://developer.mozilla.org/en-US/docs/Web/Accessibility) . Disabilities view the web-page by screen-reader and it helps them to understand the context of each picture.
Could you please add the appropriated text to each `alt`